src/emulator/auth/widget_ui.ts

```diff
@@ -339,8 +339,8 @@ button {
 
 #content {
   box-sizing: border-box;
-  width: 515px;
   margin: 16px auto;
+  max-width: 515px;
```

Comment:
Please also add a min-width to keep it usable for extreme cases (with scrolling, but at least still usable).

```suggestion
  max-width: 515px;
  min-width: 300px;
```

I think `300px` is a good cutoff since almost all mobile phones are bigger than that. See https://viewportsizes.mattstow.com/?filter=